python/src/bit.cpp

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: LGPL-3.0-or-other
+// Copyright (C) 2021 Contributors to the SLS Detector Package
+/*
+This file contains Python bindings for the RegisterAddr, BitPosition and RegisterValue
+classes.
+*/
+#include "py_headers.h"
+#include "sls/bit_utils.h"
+
+namespace py = pybind11;
+
+using sls::RegisterAddress;
+using sls::RegisterValue;
+using sls::BitPosition;
+
+void init_bit(py::module &m) {
+
+    py::class_<RegisterAddress>(m, "RegisterAddress")
+        .def(py::init())
+        .def(py::init<const std::string &>())
+        .def(py::init<uint32_t>())
+        .def(py::init<const RegisterAddress &>())
+        .def("__repr__", &RegisterAddress::str)
```

Comment:
For __repr__ do we want it to print RegisterAddress(0x45)? str should probably be only 0x45 for writing to file but the type might be useful when doing interactive stuff